scripts/pi-hole/js/messages.js

```diff
@@ -57,6 +57,22 @@ function renderMessage(data, type, row) {
         "</pre> to get the group configuration for this client."
       );
 
+    case "HOSTNAME":
+      var hint = ' '.repeat(row.blob2 + row.message.length + 2);
```

Comment:
This is also icompatible with IE

Reply:
So let's use the one-line `for` loop. This isn't a complicated thing, it doesn't warrant indentation and brackets.